node/actors/network/src/testonly.rs

```diff
@@ -13,7 +13,10 @@ use std::{
     sync::Arc,
 };
 use zksync_concurrency::{ctx, ctx::channel, io, limiter, net, scope, sync};
-use zksync_consensus_roles::{node, validator};
+use zksync_consensus_roles::{
+    node,
+    validator::{self, SecretKey},
```

Comment:
we use the convention of using qualified names for types in roles crate: things like SecretKey get easily ambiguous without knowing which role it belong to.